### DISCUSSION TITLE
The database you introspected seems to belong to a Prisma 1 project.

### DISCUSSION THREAD

**Discussion by Inayat-C** [prisma/prisma#2647] [Q&A] [answered]
Every time i run `npx prisma introspect` i get the following message:

Upgrading from Prisma 1 to Prisma 2                                   

```

The database you introspected seems to belong to a Prisma 1 project.  

Please run the following command to upgrade to Prisma 2.0: 
npx prisma-upgrade [path-to-prisma-yml] [path-to-schema-prisma]

Note: `prisma.yml` and `schema.prisma` paths are optional.

Learn more about the upgrade process in the docs:
https://pris.ly/upgrading-to-prisma2

```

The URL for the upgrade process also links to a 404. 

Running `npx prisma-upgrade` doesn't seem to solve anything either, i still get the same message when ever i run `npx prisma introspect`.

Not sure if this is because i'm doing something wrong or if it's an actual issue, any help will be appreciated!

Thanks.

**Comment by ryands17** [comment, marked as answer]
Hey @Inayat-C :wave: 
Currently you would have to run `prisma-upgrade` in the following manner:

```
npx prisma-upgrade datamodel.prisma schema.prisma
```

This is experimental so you might run into some issues. You can also read more about this [here](https://www.prisma.io/docs/guides/upgrade-from-prisma-1/should-you-upgrade#prisma-schema-upgrade-cli).

[This](https://github.com/prisma/upgrade) is the page related to `prisma-upgrade`.

This only works if you have a `datamodel.prisma` from Prisma 1. Is that so or it's just a Prisma 1 DB?

**Reply by Inayat-C** [reply]
To be honest with you, i don't know what a Prisma 1 DB is. 

I'm running the local default postgres database (set to my system user credentials) and set up prisma following this guide from the docs: https://www.prisma.io/docs/getting-started/setup-prisma/add-to-existing-project-node-postgres

**Reply by ryands17** [reply]
Could you share your DB dump if possible? I would need to run intospect on it to reproduce.

**Reply by Inayat-C** [reply]
[db_dump.txt](https://github.com/prisma/prisma/files/4722613/db_dump.txt)

**Reply by ryands17** [reply]
@Inayat-C I have run the dump and introspected the DB. The warning you have mentioned above occurs and but the models are being created properly.

You will be able to see the models in you `schema.prisma` file.

**Reply by Inayat-C** [reply]
Yes, that was my concern. I know the models are being created properly, but the warning still persists so wanted to know what is actually causing the warning to appear? Thanks for all the help thus far @ryands17 !

**Reply by ryands17** [reply]
Yes this is an issue and we are looking into this. Thanks for reporting :)

**Reply by Inayat-C** [reply]
Ah, that's fine then nothing else to worry about from my end. Thanks for all the help!

**Comment by yazer79** [comment]
I have a fresh installation of prisma 2.19.0 and i am getting same error when introspecting a simple test db with 2 columns
"The database you introspected seems to belong to a Prisma 1 project."
I do not have any previous prisma versions, or a prisma yml

**Reply by janpio** [reply]
That sounds like a false positive. Can you post your current Prisma schema please? Thanks.

**Comment by ludralph** [comment]
Hi @Inayat-C,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that have already been marked as answered but remain open.

If this discussion still requires further input or clarification, feel free to reopen it or start a new one with updated details. Your contributions are invaluable to the community, and we’re here to help!

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and ongoing support of the Prisma community!
